it-test/src/test/java/io/knotx/forms/test/integration/FormsIntegrationTest.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (C) 2018 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.knotx.forms.test.integration;
+
+import static io.knotx.junit5.util.RequestUtil.subscribeToResult_shouldSucceed;
+
+import io.knotx.dataobjects.ClientRequest;
+import io.knotx.dataobjects.ClientResponse;
+import io.knotx.dataobjects.Fragment;
+import io.knotx.dataobjects.KnotContext;
+import io.knotx.junit5.KnotxApplyConfiguration;
+import io.knotx.junit5.KnotxExtension;
+import io.knotx.reactivex.proxy.KnotProxy;
+import io.reactivex.Single;
+import io.reactivex.functions.Consumer;
+import io.vertx.core.json.JsonObject;
+import io.vertx.junit5.VertxTestContext;
+import io.vertx.reactivex.core.MultiMap;
+import io.vertx.reactivex.core.Vertx;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.List;
+import org.junit.Ignore;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@Ignore
+@ExtendWith(KnotxExtension.class)
+public class FormsIntegrationTest {
+
+  private final static String CORE_MODULE_EB_ADDRESS = "knotx.knot.forms";
+  private final static String FORM_NAME = "someId456";
+
+  @Test
+  @KnotxApplyConfiguration("formsStack.conf")
```

Comment:
I think we can define the example configuration in `conf` folder:
```
conf/
       formStack.conf
       includes/
                    formsKnot.conf
                    exampleFormAdatper.conf 
```
and then in the test method specify `formsStack.conf` that contains default settings and add `testScenarioXXX.conf` that overrides defaults. The example can be found here: io.knotx.example.monolith.SampleApplicationIntegrationTest#whenRequestingLocalSimplePageWithGetCustomSymbol_expectLocalSimpleHtml.

Additionally we can specify ExampleFormAdapter behaviour based on expected scenarios. So the configuration can be:

exampleFormAdapter.address = ...
exampleFormAdapter.testStrategy = GET_REQUEST

And in the Example Form Adapter we can implement all cases. Please not that both Example Form Adapter and `testScenarioXXX.conf` are defined in tests scope.